src/khoj/interface/web/content_type_input.html

```diff
@@ -106,28 +106,38 @@ <h2 class="section-title">
 
     submit.addEventListener("click", function(event) {
         event.preventDefault();
-        let globFormat = "**/*."
+        let globFormat = "**/*"
         let suffixes = [];
         if ('{{content_type}}' == "markdown")
-            suffixes = ["md", "markdown"]
+            suffixes = [".md", ".markdown"]
         else if ('{{content_type}}' == "org")
-            suffixes = ["org"]
+            suffixes = [".org"]
         else if ('{{content_type}}' === "pdf")
-            suffixes = ["pdf"]
+            suffixes = [".pdf"]
         else if ('{{content_type}}' === "plaintext")
-            suffixes = ['*']
+            suffixes = ['*', '*.*']
```

Comment:
Shouldn't the plaintext suffix just be `.*`, so that the plaintext glob will look like `**/*.*` (like it does currently)?
 
```suggestion
            suffixes = ['.*']
```

Reply:
Hey @spott! I'm going to commit @debanjum 's suggestion because I want this to make it for our next release. If you do have time to look at it again and remember why you configured the plaintext pattern in this way, don't hesitate to leave a comment.